src/Main.java

```diff
@@ -1,5 +1,350 @@
+        //
+        //С помощью цикла while посчитайте, сколько месяцев потребуется, чтобы накопить 2 459 000 рублей при условии,
+        // что первоначально мы имеем 0 рублей и готовы откладывать по 15 тысяч рублей.
+        //
+        //Результат программы должен быть выведен в консоль с тем количеством месяцев, которое необходимо для накопления
+        // данной суммы. Формат сообщения: «Месяц …, сумма накоплений равна … рублей» .
+        int i;
+        int savingsPerMonth = 15000;
+        int totalInBank = 0;
+        for (i = 1; i <= 12; i++) {
+            totalInBank = totalInBank + totalInBank / 100;
+            totalInBank = totalInBank + savingsPerMonth;
+            System.out.println("Копим " + i + " месяцев. " + "В банке 'ЗверБанк' сейчас: " + totalInBank);
+        }
+        System.out.println("_________________________________________________________________________________________");
+        System.out.println("______________________________________Задача 2___________________________________________");
+        System.out.println("_________________________________________________________________________________________");
+        //Задача 2
+        //Выведите в одну строку через пробел числа от 1 до 10 с помощью цикла while.
+        //
+        //На следующей строке выведите числа в обратном порядке от 10 до 1 с помощью цикла for.
+        //
+        //Для обоих циклов можно использовать как одну общую переменную, так и свою в каждом цикле.
+        //
+        //Не забудьте выполнить переход на новую строку между двумя циклами.
+        //
+        //В результате программы вывод должен получиться следующим:
+        //
+        //1 2 3 4 5 6 7 8 9 10
+        //
+        //10 9 8 7 6 5 4 3 2 1
+
+        for (i = 1; i <= 10; i++) {
+            System.out.print(i + " ");
+        }
+        System.out.println();
+        for (i = 10; i >= 0; i--) {
+            System.out.print(i + " ");
+        }
+
+        System.out.println("_________________________________________________________________________________________");
+        System.out.println("______________________________________Задача 3___________________________________________");
+        System.out.println("_________________________________________________________________________________________");
+        //Задача 3
+        //В стране Y население — 12 миллионов человек.
+        //
+        //Рождаемость составляет 17 человек на 1000, смертность — 8 человек. Рассчитайте, какая численность населения
+        // будет через 10 лет, если показатели рождаемости и смертности постоянны.
+        //
+        //В консоль выведите результат операции на каждый год в формате: «Год …, численность населения составляет …».
+        int population = 12_000_000;
+        int birthRate = 17;
+        int deathRate = 8;
+        int yearCount = 2024;
+
+        while (yearCount <= 2034) {
```

Comment:
откуда 2034? не надо в уме никогда ничего считать
```suggestion
        while (yearCount <= yearCount + 10) {
```